src/utils/LinkedInPage.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useLinkedIn } from "react-linkedin-login-oauth2";
+import { getAccessTokenData, getMemberDetails } from "./LinkedInUtils";  
+
+const LinkedInPage = () =>{
+
+  // get the user authorization token from linkedin login
+  const{ linkedInLogin } = useLinkedIn({
+    clientId: process.env.REACT_APP_LINKEDIN_CLIENT_ID,
+    redirectUri: `${window.location.origin}/linkedin/callback`,
+    onSuccess: (authCode) => {
+      console.log("AuthCode: ",authCode);
+      setAuthCode(authCode);
+      setErrorMessage("");
+    },
+    scope: ["w_member_social","openid","profile","email"],
+    onError: (error) => {
+      console.log(error);
+      // setCode("");
+      setErrorMessage(error.errorMessage);
+    },
+  });
+
+
+  const [authCode, setAuthCode] = React.useState("");
+  const [errorMessage, setErrorMessage] = React.useState("");
+  let accessTokenData;
```

Comment:
Why use let?

Reply:
Its not necessary anymore. I have removed it in the new commit and will push it today.